docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -445,18 +444,22 @@ There are still five PersistentVolumeClaims and five PersistentVolumes.
 When exploring a Pod's [stable storage](#stable-storage), we saw that the 
 PersistentVolumes mounted to the Pods of a StatefulSet are not deleted when 
 the StatefulSet's Pods are deleted. This is still true when Pod deletion is 
-caused by scaling the StatefulSet down. This feature can be used to facilitate 
-upgrading the container images of Pods in a StatefulSet.
+caused by scaling the StatefulSet down. 
 
-## Updating Containers
-As demonstrated in the [Scaling a StatefulSet](#scaling-a-statefulset) section,
-the `replicas` field of a StatefulSet is mutable. The only other field of a 
-StatefulSet that can be updated is the `spec.template.containers` field. 
+## Updating StatefulSets
 
-StatefulSet currently *does not* support automated image upgrade. However, you 
-can update the `image` field of any container in the podTemplate and delete 
-StatefulSet's Pods one by one, the StatefulSet controller will recreate 
-each Pod with the new image.
+In Kubernetes 1.7, the StatefulSet controller supports automated updates.  The 
+strategy used is determined by the `spec.updateStrategy` field of the 
+StatefulSet API Object. This feature can be used to upgrade the container 
+images, resource requests and/or limits, labels, and annotations of the Pods in a 
+StatefulSet. There are two valid update strategies, `OnDelete` and 
```

Comment:
The tutorial can be opinionated about what features it uses.  I'd suggest just saying "In this example we use `spec.updateStrategy.rollingUpdate`" and link to the concept guide for more info on the options.  You can then drop the whole `### On Delete` section.